validate.py

```diff
@@ -121,7 +121,7 @@ def determine_if_winner():
   points_so_far = 0
   for user, user_points in util.get_user_points().items():
     if rnd < 0.00001 * (util.total_user_points(user_points) + points_so_far):
-      raise Exception('%s wins!' % user)
+      raise Exception('%s wins! They had a %s chance to win.' % (user, user_points / 100000))
```

Comment:
I think this would be tidier as how many points each user has and how likely they are to win:

```
Points:
  jburger424: 11 (0.005%)
    bonus: 10
    merge: 1
  jeffkaufman: 100015 (49.996%)
    bonus: 100010
    merge: 5
  pavellishin: 100012 (49.994%)
    bonus: 100010
    merge: 2
  tnelling: 10 (0.005%)
    bonus: 10
    merge: 0
```

Or, since that would be a lot of refactoring, just print:

```
Probability of winning:
  jburger424: 0.005%
  jeffkaufman: 49.996%
  pavellishin: 49.994%
  tnelling: 0.005%
```

Reply:
Hmm... do we want this to display all the time, or only when someone won?

Reply:
Actually, it's still a lot of refactoring either way, if only because you can't print all the percentages unless the last player wins.

Reply:
I think we should do it all the time if we're going to do it.

But it's not a major refactor once #231 is in:

```
print('Probability of winning:')
for user, user_points in summed_user_points:
  print('%s: %.3f%%' % (user, user_points * scalar * 100))
```

Reply:
True, let's just get that in first.